source/features/releases-tab.tsx

```diff
@@ -110,6 +110,11 @@ async function init(): Promise<false | void> {
 			})
 		);
 
+		// Hide redundant 'Releases' section from repo sidebar
+		if (pageDetect.isRepoRoot()) {
+			select('.BorderGrid-cell a[href$="/releases"]')?.closest('.BorderGrid-row')!.setAttribute('hidden', '');
```

Comment:
This appears to always exist (so no `?.`) but it might not be ready yet, the previous code only awaits the header. I think you need:
```suggestion
			const sidebarReleases = await elementReady('.BorderGrid-cell a[href$="/releases"]');
			sidebarReleases!.closest('.BorderGrid-row')!.hidden = true;
```